crates/holochain/build.rs

```diff
@@ -131,10 +109,9 @@ mod version_info {
     /// This will be used populate the VERSION_INFO environment variable,
     /// which will be displayed as JSON when `holochain --version-info` is called.
     pub(crate) fn populate_env() {
-        println!(
-            "cargo:rustc-env=BUILD_INFO={}",
-            BuildInfo::retrieve().as_json_string()
-        );
+        let json = BuildInfo::retrieve().as_json_string();
+        println!("cargo:rustc-env=BUILD_INFO={}", json);
+        //println!("cargo:warning={}", json);
```

Comment:
leftover

Reply:
i'd like to keep this in, so we can easily debug it in the future... but we don't want it to be a real warning.
```suggestion

        // incase you want to debug the output:
        //println!("cargo:warning={}", json);
```